### PR TITLE
Release 2.0.9 merge see all

### PR DESCRIPTION
**Pull request by asogwachiuba**
Successfully merged.

### CODE REVIEW

File: lib/flavors.dart [hunk 1]
```diff
@@ -39,7 +39,7 @@ class F {
   static String get placeBaseUrl {
     switch (appFlavor) {
       case Flavor.prod:
-        return "https://places.ibloov.com";
+        return "https://ibloov-places-api.herokuapp.com";
```
Comment (divincoder) [inline lib/flavors.dart:42]: why? revert this change

**Review by divincoder — APPROVED** [review]

File: lib/flavors.dart (hunk 1 above)
Comment (asogwachiuba) [reply lib/flavors.dart:42]: It was affecting the api request while testing. I will change it now. Just want abit of clarity, is it expected to affect apicalls while testing or there was something i should have done that i didn't do to avert it

Comment (divincoder) [reply lib/flavors.dart:42]: I don't understand. Were you running the app in Prod instead of stage?
